src/styles/fonts/_fontawesome.scss

```diff
@@ -0,0 +1,2199 @@
+/*!
```

Comment:
forgive me, I made a mistake. I thought we already had fontawesome in the code base but we don't Let's go back to what you had earlier with the images. I am so sorry again!